src/lambda/update-syllabus/index.py

```diff
@@ -0,0 +1,81 @@
+import json
+import boto3
+import logging
+import os
+from boto3.s3.transfer import S3Transfer
+from const import *
+from utils import *
+
+
+s3 = boto3.client('s3')
+s3r = boto3.resource('s3')
+transfesr = S3Transfer(s3)
+
+dynamodb = boto3.resource('dynamodb')
+table    = dynamodb.Table('syllabus')
+    
+    
+def compare_syllabus(now_name,old_name,school):
+    old_dict = {}
+    with open(old_name) as f:
```

Comment:
try an expressive variable name instead of f

Reply:
Used old_file as the variable name.